operator-common/src/main/java/io/strimzi/operator/common/Util.java

```diff
@@ -858,4 +862,32 @@ public static boolean isMaintenanceTimeWindowsSatisfied(Reconciliation reconcili
     public static String encodeToBase64(String encode)  {
         return Base64.getEncoder().encodeToString(encode.getBytes(StandardCharsets.US_ASCII));
     }
+
+    /**
+     * Validate container's common compute resources.
```

Comment:
Plase explin what exactly you validate. You seem to validate some specific use cases - but there are other things validated elsewhere and possibly other unvalidated things. So we should be clear about what it does. Maybe you can also make it more clear in the method name that the focus is on validating limits against requests.